src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,89 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="console.command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
Also commands are unlikely to be referenced after they have been registered. So the service ids don't really matter.